swagger2locustio/templates/locustfile_templates.py

```diff
@@ -1,8 +1,7 @@
 from jinja2 import Template
 
 
-file_template = Template("""
-import os
+file_template = Template("""import os
```

Comment:
tabulation is very important in the case of generation locustfile. have you checked file generation after changing this line?

Reply:
yes, I've checked it